src/index.js

```diff
@@ -10,7 +11,6 @@ import {
   allFilesAccepted,
   fileMatchSize,
   onDocumentDragOver,
-  getDataTransferItems as defaultGetDataTransferItem,
```

Comment:
This removes an ability to plug different file parsers. Why?

Reply:
It does not. Users can still use their own thing. The only difference is that now we use the file-selector fn by default.